### PR TITLE
Studio: mention live.end() in live metrics docs

### PR DESCRIPTION
**Pull request by tapadipti**
Address https://github.com/iterative/dvc.org/pull/4242#issuecomment-1385937862

### CODE REVIEW

File: content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md [hunk 1]
```diff
@@ -81,14 +81,13 @@ with Live() as live:
...
 ```
 
-<admon type="warn">
+<admon>
 
-If `live.end()` is not called, Iterative Studio will continue to wait for new
-updates forever, because it will not receive the signal that the experiment has
-ended.
+DVCLive signals the end of the experiment using `live.end()`. Using
+`with Live() as live:` ensures that `live.end()` is automatically called when
```
Comment (daavoo) [inline content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md:89]:
```suggestion
`with Live() as live:` or one of the DVCLive integrations for [ML Frameworks](doc/dvclive/api-reference/ml-frameworks) ensures that `live.end()` is automatically called when
```

File: content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md [hunk 2]
```diff
@@ -81,14 +81,13 @@ with Live() as live:
...
+the experiment concludes.
```
Comment (daavoo) [inline content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md:90]:
```suggestion
DVCLive signals the end of the experiment using `live.end()`. Using
`with Live() as live:` r one of the DVCLive integrations for 
[ML Frameworks](doc/dvclive/api-reference/ml-frameworks) ensures 
that `live.end()` is automatically called when the experiment concludes.
```

File: content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md [hunk 3]
```diff
@@ -81,14 +81,14 @@ with Live() as live:
...
 
-<admon type="warn">
+<admon>
 
-If `live.end()` is not called, Iterative Studio will continue to wait for new
-updates forever, because it will not receive the signal that the experiment has
-ended.
+DVCLive signals the end of the experiment using `live.end()`. Using
+`with Live() as live:` or one of the DVCLive integrations for
+[ML Frameworks](doc/dvclive/api-reference/ml-frameworks) ensures that
```
Comment (daavoo) [inline content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md:90]:
```suggestion
[ML Frameworks](/doc/dvclive/api-reference/ml-frameworks) ensures that
```

**Review by daavoo — APPROVED** [review]

**Review by dberenbaum — APPROVED** [review]
Thanks @tapadipti, LGTM!

**Pull request merged**